packages/hash/api/src/model/knowledge/account.fields.ts

```diff
@@ -0,0 +1,77 @@
+import { OrgModel, UserModel } from "..";
+import { GraphApi } from "../../graph";
+import {
+  generateSchemaBaseUri,
+  generateWorkspacePropertyTypeSchema,
+  RESTRICTED_SHORTNAMES,
+  workspaceTypesNamespaceUri,
+} from "../util";
+
+module AccountId {
+  // Generate the schema for the account id property type
+  export const accountIdPropertyType = generateWorkspacePropertyTypeSchema({
+    title: "Account ID",
+    possibleValues: [{ primitiveDataType: "Text" }],
+  });
+
+  export const accountIdBaseUri = generateSchemaBaseUri({
+    namespaceUri: workspaceTypesNamespaceUri,
+    kind: "propertyType",
+    title: accountIdPropertyType.title,
+  });
+}
+
+module Shortname {
```

Comment:
I'm using `module`s to group plain old functions, I am not sure if it's an anti-pattern in typescript, but this seems to be the way to do it instead of using `namespace`s

Reply:
As discussed offline, this seems to be very unused and therefore not something we want to do. The better approach is to use file modules instead.

Reply:
Removed in 61042384341d6fc0f1ddbd3bc77ee74bd3c3c1de. Ended up not making any new files as we weren't really using the logical grouping